resource/src/main/java/com/epam/learn/resource/repository/ResourceRepository.java

```diff
@@ -0,0 +1,15 @@
+package com.epam.learn.resource.repository;
+
+import com.epam.learn.resource.domain.ResourceLocation;
+import org.springframework.data.repository.CrudRepository;
+import org.springframework.stereotype.Repository;
+
+import java.util.List;
+
+@Repository
+public interface ResourceRepository extends CrudRepository<ResourceLocation, Integer> {
+
+    List<ResourceLocation> deleteByIdIn(List<Integer> ids);
```

Comment:
I would keep it as `deleteByIds`.

Reply:
`deleteByIds` is not acceptable method name, when using Repository interfaces, cause it will search for Entity's property named `Ids` which is not the case here. So I kept existing naming

Reply:
It is OK then.